core_lib/src/managers/playbackmanager.h

```diff
@@ -84,7 +84,7 @@ class PlaybackManager : public BaseManager
     int mMarkOutFrame = 10;
     int mActiveSoundFrame = 0;
 
-    int mFps = 12;
+    int mFps;
```

Comment:
Why remove the default value? this could lead to undefined behaviour in worst case scenario.